patterns/stories-podcast-banner.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * Title: Stories Podcast Hero
+ * Slug: twentytwentyfive/stories-podcast-hero
+ * Categories: featured, hero
+ * Keywords: podcast, hero, stories
+ */
+?>
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|70","bottom":"var:preset|spacing|70","left":"var:preset|spacing|70","right":"var:preset|spacing|70"}},"color":{"background":"#F6CFF4"}},"layout":{"type":"constrained"}} -->
```

Comment:
A few adjustments to this group:

- We need to have this group alignment to be full-width.
- The background color should be coming from the `accent-2` preset.
- The spacing needs to be adjusted to what the design is presenting. The left/right paddings should be set to default.